### ISSUE TITLE
[BUG] Marble rolling pitch gets progressively lower

### ISSUE THREAD

**Issue by thearst3rd** [MBU-Team/OpenMBU#95]
As you play the game, the marble rolling pitch gets progressively lower. It seems to happen every time you exit or complete a level. I'm not sure if those are the only times it happens, but I know those are consistent ways to reproduce it. Restarting a level from the menu (_not_ exiting and reentering) does not seem to trigger the bug.

Here is a video of me reproducing the bug by exiting a level and reentering it multiple times:

https://www.youtube.com/watch?v=2Kdx3s8pzYY

**Comment by thearst3rd** [comment]
It turns out this bug doesn't just make the rolling pitch get lower - it can also get higher. Here is a video of a few clips I put recorded of the pitch getting higher as I play through the levels.

https://www.youtube.com/watch?v=85M1UH1F_-w

I've started looking into the issue, and I've made an interesting discovery. I've hard coded the pitch of the marble rolling sound to always be constant, and even still, by exiting the level I can cause the marble pitch to get higher or lower. I changed the following line:

https://github.com/MBU-Team/OpenMBU/blob/87b8a42ba78efb67a978c731cb0817799400033d/engine/source/game/marble/marble.cpp#L1399

into `mRollHandle->setPitch(1.0f);`.

I'm looking into it more now.
